### PR TITLE
Adds Fennecs and Foxes.

### LINKED ISSUE #698: AMI RIGs useless for xenoarch/anom
Issue made on 5.5.2016

The AMI hardsuit modules cannot be deployed by xeno archaeologists nor anomalists due to a lack of access on their part. 
Image evidence (to look nice); image of evidence

### LINKED ISSUE #702: Icon updating for cyborg items is broken (combat droid)
Combat droids no longer update icons when in shield mode or mobility mode. This is likely a result of a recent update to cyborgs changing how they handle icons.

### PR DESCRIPTION
**Pull request by Cameron653**
# Adds Fennecs and Foxes
#683 

**Fennec and fox sprites**: http://puu.sh/oGiq8/a3c19af398.png

**Fennecs**: Get hot slower, get colder faster.

**Foxes**: Get cold slower, get hot faster.

**Fox tail**: http://puu.sh/oGj7R/d3b9848396.png

**Fennec tail**: http://puu.sh/oGj8W/5fe4340da0.png

**Fox ears (MODIFIED)**: http://puu.sh/oGzpy/0bb65a69cc.png

**Fennec ears**: http://puu.sh/oGjac/18f9f75787.png

(Sprites created by: ArgobargSoup)

**Note**: They currently have little/no blurb text, AKA the lore. They also don't have a custom language, but both have the Siik'tajr language (Tajaran.)

Foxes and Fennecs both have tajaran spacesuit sprites, since Tajaran are the closest race to them. Making individual spacesuit sprites for them would take a long time, and I'm not a spriter. The fox and fennec

I'm not an amazing lore-writer, so feel free to add actual lore to their blurbs. I can provide a custom language later, but having too many languages in play at once might not be a good thing.

~~**NOTE V2:** Also fixes #702  And fixes #698~~ MOVED

### CODE REVIEW

**Comment by Cameron653** [comment]
Adding DNM. Modifying something major.

**Comment by antsnap** [comment]
Nice. Good job!

**Comment by SpadesNeil** [comment]
When you're done tweaking, add the tag "Needs Testing"

**Comment by Cameron653** [comment]
I think I've got everything. The fox ears work, the fennec ears work, etc etc.

Only one thing, and that is that the fox hair colored ears are a bit bright and a bit off, but that's hardly a problem.

**Comment by SpadesNeil** [comment]
I noticed there's no human/mob defined. Bird people also have this problem currently.

File: code/modules/mob/living/carbon/human/human_species.dm [hunk 1]
```diff
@@ -36,4 +36,13 @@
 	..(new_loc, "Sergal")
 
 /mob/living/carbon/human/slimePerson/New(var/new_loc)
-	..(new_loc, "Slime Person")
\ No newline at end of file
+	..(new_loc, "Slime Person")
+
+/mob/living/carbon/human/nevrean/New(var/new_loc)
```
Comment (Cameron653) [inline code/modules/mob/living/carbon/human/human_species.dm]: All of this also fixes #437

File: code/modules/clothing/spacesuits/void/void.dm [hunk 2]
```diff
@@ -12,20 +12,24 @@
 	//Species-specific stuff.
 	species_restricted = list("exclude","Unathi","Tajara","Akula","Sergal","Nevrean","Diona","Vox", "Xenomorph", "Xenomorph Drone", "Xenomorph Hunter", "Xenomorph Sentinel", "Xenomorph Queen")
 	sprite_sheets_refit = list(
-		"Unathi" = 'icons/mob/species/unathi/helmet.dmi',
-		"Tajara" = 'icons/mob/species/tajaran/helmet.dmi',
-		"Skrell" = 'icons/mob/species/skrell/helmet.dmi',
-		"Akula"  = 'icons/mob/species/akula/helmet.dmi',
-		"Sergal" = 'icons/mob/species/sergal/helmet.dmi',
+		"Unathi"  = 'icons/mob/species/unathi/helmet.dmi',
```
Comment (Cameron653) [inline code/modules/clothing/spacesuits/void/void.dm]: Also, the reason I put spaces infront of all of this was to make it better looking. Doesn't actually effect the code.

**Comment by Cameron653** [comment]
ALRIGHT! I think I got everything working.
Fennec and foxes both have separate spacesuits that can be obtained by the suit cycler (Same sprite as tajarans)
Their tail is automatically selected, but their ears still have to be manually picked.
Added them both (And the Nevrean) to human_species.dm
All other clothings work normally (That I could see.)

Going to remove the DNM tag, as I've done everything requested (Except for the primitive species, but that's not a huge problem, and I'm not a spriter.) and it seems to be stable and ready to be tested and merged.

**Comment by SpadesNeil** [comment]
Should probably keep bugfixes separate.

**Comment by SpadesNeil** [comment]
Also putting back the DNM tag only because we're not calling them straight up "foxes" and "Fennecs" because they should have some ACTUAL LORE to accompany them.

**Comment by Cameron653** [comment]
Ah, good idea. I'll keep an eye on the forum thread and add information into the code as it comes in/when the lore is finalized

**Comment by Cameron653** [comment]
Fixed #698 and a fox and fennec rigsuit related problem

**Comment by Cameron653** [comment]
"Should probably keep bugfixes separate in future."

Yeah. I could probably take all the bugfixes out and upload it on my second account as a pure bugfix PR if you'd like.

**Comment by Cameron653** [comment]
Any news on this? As in, what should I input for their name and lore?

**Comment by SpadesNeil** [comment]
We need a wiki page for them first, and a wiki entry. Even if it has no pictures yet.

We will call them Zorren though. Fennecs will be "Flatland" Zorren, and the Foxes will be "Highlander" Zorren. They are indigenous to Virgo-Prime.

**Comment by Cameron653** [comment]
Names changed. Lore made into placeholders. Everything works.

**Comment by SpadesNeil** [comment]
I said Zorren, not Voxxen. :P We already have a species called Vox.

**Comment by Cameron653** [comment]
Ack, I should get more sleep when I code.  Will fix it in a bit.

**Comment by Cameron653** [comment]
Alright. I fixed the names again. This time I made sure to get it right.

**Comment by SpadesNeil** [comment]
https://wiki.vore-station.net/Backstory They still need an entry on this page before I merge these.

**Comment by SpadesNeil** [comment]
Okay, wiki entry is made. We can merge this now unless you have any last minute changes.

**Comment by Cameron653** [comment]
If everything looks good codewise, feel free to merge it.

**Comment by killer653** [comment]
Since I can't edit my comment above, last I tested everything worked.

**Comment by Cameron653** [comment]
Snuck in a quick fix. The sergal spacesuit sprite for ninja rigs wasn't correct and was missing a west sprite with tail.

**Pull request merged**